mbed_app.json

```diff
@@ -1,8 +1,22 @@
 {
+    "config": {
+        "hostname": {
+            "help": "The demo will try to connect to this web address on port 80.",
+            "value": "\"ifconfig.io\""
+        }
+    },
     "target_overrides": {
         "*": {
-            "platform.stdio-convert-newlines": true,
-            "target.network-default-interface-type": "ETHERNET"
+            "nsapi.default-wifi-security": "WPA_WPA2",
+            "nsapi.default-wifi-ssid": "\"YOUR_SSID\"",
+            "nsapi.default-wifi-password": "\"YOUR_PASSWORD\"",
+            "platform.stdio-baud-rate": 115200
+        },
+        "DISCO_F413ZH": {
+            "target.components_add": ["ism43362"]
+        },
+        "DISCO_L475VG_IOT01A": {
+            "target.components_add": ["ism43362"]
```

Comment:
@MarceloSalazar Do you know why this component is not part of Mbed OS ?

Reply:
We haven't see this as a requirement. 
Other examples include a configuration and .lib file which can be imported and compiled for a specific targets - see this:
https://github.com/ARMmbed/mbed-os-example-pelion/blob/master/mbed_app.json#L303

Reply:
Thanks for the clarification, I remember it was an ask for the BLE driver on this board to be part of the OS because it is on the board. I was just surprised the same ask hasn't be formulated for WiFi. 
Thanks for your answer @MarceloSalazar .